src/pyggdrasil/tree_inference/_huntress.py

```diff
@@ -39,7 +41,12 @@ def huntress_tree_inference(
     assert 0 <= false_positive_rate < 1
     assert 0 <= false_negative_rate < 1
 
-    n_cells, n_mutations = mutations.shape
+    # check that all entries are either 0 or 1 else throw error
+    if not np.all(np.logical_or(mutation_mat == 0, mutation_mat == 1)):
+        raise Exception("Huntress does not allow with missing data.")
```

Comment:
Nice catch! If I were, to complain about: it's a good practice to use as precise exception types as possible, i.e., `ValueError` in this case, rather than a general `Exception` :wink:

Reply:
Oh, yes of course. Thanks for pointing out.